<ui version="4.0" >
 <class>httpArgsDlg</class>
 <widget class="QDialog" name="httpArgsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>525</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>525</width>
    <height>312</height>
   </size>
  </property>
  <property name="font" >
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>HTTP(S) Options - aria2fe</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources.qrc" >:/pixmaps/pixmaps/download_22x22.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox1" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>154</y>
     <width>241</width>
     <height>87</height>
    </rect>
   </property>
   <property name="title" >
    <string>HTTP User</string>
   </property>
   <widget class="QLabel" name="txtLabel4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>User ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtLabel5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>51</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Passwd:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="httpUserEdit" >
    <property name="geometry" >
     <rect>
      <x>67</x>
      <y>20</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="httpPasswdEdit" >
    <property name="geometry" >
     <rect>
      <x>67</x>
      <y>51</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="httpContinueCkBox" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>32</y>
     <width>176</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Continue Download</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtLabel2" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>92</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Auto Rename:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="httpRenameCombo" >
   <property name="geometry" >
    <rect>
     <x>149</x>
     <y>92</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>true</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>false</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox2" >
   <property name="geometry" >
    <rect>
     <x>272</x>
     <y>154</y>
     <width>241</width>
     <height>87</height>
    </rect>
   </property>
   <property name="title" >
    <string>HTTP Proxy</string>
   </property>
   <widget class="QLabel" name="txtLabel10" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>User ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtLabel11" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>51</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Passwd:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="httpProxyUserEdit" >
    <property name="geometry" >
     <rect>
      <x>67</x>
      <y>20</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="httpProxyPasswdEdit" >
    <property name="geometry" >
     <rect>
      <x>67</x>
      <y>51</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="httpSchemeCombo" >
   <property name="geometry" >
    <rect>
     <x>413</x>
     <y>32</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>basic</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="txtLabel8" >
   <property name="geometry" >
    <rect>
     <x>272</x>
     <y>92</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Proxy Method:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="httpMethodCombo" >
   <property name="geometry" >
    <rect>
     <x>413</x>
     <y>92</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>tunnel</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>get</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="txtLabel6" >
   <property name="geometry" >
    <rect>
     <x>272</x>
     <y>32</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Authentication Scheme:</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtLabel9" >
   <property name="geometry" >
    <rect>
     <x>272</x>
     <y>122</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Proxy H:P:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="httpProxyServerEdit" >
   <property name="geometry" >
    <rect>
     <x>345</x>
     <y>122</y>
     <width>169</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="txtLabel3" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>122</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>User Agent:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="httpAgentEdit" >
   <property name="geometry" >
    <rect>
     <x>81</x>
     <y>122</y>
     <width>169</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="upperFrame" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>505</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth" >
    <number>2</number>
   </property>
  </widget>
  <widget class="QFrame" name="lowerFrame" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>251</y>
     <width>505</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth" >
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn" >
   <property name="geometry" >
    <rect>
     <x>437</x>
     <y>272</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn" >
   <property name="geometry" >
    <rect>
     <x>352</x>
     <y>272</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreBtn" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>272</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Restore Defaults</string>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/pixmaps/pixmaps/refresh.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="txtLabel1" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>62</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Enable Persistence:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="httpPersistenceCombo" >
   <property name="geometry" >
    <rect>
     <x>149</x>
     <y>62</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>true</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="txtLabel7" >
   <property name="geometry" >
    <rect>
     <x>272</x>
     <y>62</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Enable Pipelining:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="httpPipeliningCombo" >
   <property name="geometry" >
    <rect>
     <x>413</x>
     <y>62</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>false</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>true</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>httpContinueCkBox</tabstop>
  <tabstop>httpSchemeCombo</tabstop>
  <tabstop>httpPersistenceCombo</tabstop>
  <tabstop>httpPipeliningCombo</tabstop>
  <tabstop>httpRenameCombo</tabstop>
  <tabstop>httpMethodCombo</tabstop>
  <tabstop>httpAgentEdit</tabstop>
  <tabstop>httpProxyServerEdit</tabstop>
  <tabstop>httpUserEdit</tabstop>
  <tabstop>httpPasswdEdit</tabstop>
  <tabstop>httpProxyUserEdit</tabstop>
  <tabstop>httpProxyPasswdEdit</tabstop>
  <tabstop>okBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
  <tabstop>restoreBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc" />
 </resources>
 <connections/>
</ui>
